### ISSUE TITLE
NuGet package name for NLog support

### ISSUE THREAD

**Issue by IanYates** [fo-dicom/fo-dicom#39]
I suggest we rename the NLog package for fo-dicom.  If you search in NuGet for fellow oak, only the main package and the Serilog support appear.
![capture](https://cloud.githubusercontent.com/assets/1314477/8519921/675dfd78-2419-11e5-828e-191a163fe7bf.png)

That's because the NLog package is named differently. Searching for dicom finds the main package, Serilog support and the NLog support
![capture](https://cloud.githubusercontent.com/assets/1314477/8519957/8de3a376-2419-11e5-9642-8b4cfef70672.png)

The package Id itself can remain as is, as it's consistent with the other packages.  Thoughts?

**Comment by anders9ustafsson** [comment]
I completely agree, and Ihave actually already made the change. I modified the titles of both logging packages *after* I published the NLog package but *before* I published the Serilog package, hence the inconsistency. When the official releases are published, naming will be consistently Fellow Oak DICOM. 

I don't have access to my workstation for another week, so my responses may be a little sparse for a while. Just to give everyone else the chance to have their say as well :-)
